RadarSDK/Radar.m

```diff
@@ -711,6 +711,93 @@ + (void)getMatrixFromOrigins:(NSArray<CLLocation *> *_Nonnull)origins
                                         }];
 }
 
++ (void)sendEvent:(NSString *)name
+     withMetadata:(NSDictionary *_Nullable)metadata
+completionHandler:(RadarSendEventCompletionHandler)completionHandler {
+    [self trackOnceWithCompletionHandler:^(RadarStatus status, CLLocation * _Nullable location, NSArray<RadarEvent *> * _Nullable events, RadarUser * _Nullable user) {
+        if (status != RadarStatusSuccess) {
+            if (completionHandler) {
```

Comment:
The `RadarSendEventCompletionHandler` is marked as `NONNULL`, so we don't need these `if (completionHandler)` checks.

Reply:
makes sense, prob a cleanup item within a larger refactor as this mismatch exists elsewhere in the SDK, but out of scope for this PR